dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/publish/servers/io.openliberty.jaxrs.fat.ejbsubresource/server.xml

```diff
@@ -0,0 +1,11 @@
+<server>
+    <featureManager>
+        <feature>componenttest-1.0</feature>
+        <feature>jaxrs-2.0</feature>
+        <feature>servlet-3.1</feature>
+        <feature>ejb-3.2</feature>
```

Comment:
The bucket you put this in has cdi in its name but the server does not have cdi feature enabled.  Should it?

Reply:
no, I put it here because we haven't moved the EJB tests from WS-CD-Open and the main bucket is getting too full.